env/index.js

```diff
@@ -0,0 +1,7 @@
+import dotenv from 'dotenv'
+
+dotenv.config()
+
+export default {
+  databaseurl: process.env.DATABASE_URL || 'mongodb://127.0.0.1:27017/smart-offer'
```

Comment:
Change this to `databaseUrl`; remember to follow up `camelCase`

Reply:
Fixed